tests/hpc/munge_master.pm

```diff
@@ -9,7 +9,7 @@
 
 # Summary: Installation of munge package from HPC module and sanity check
 # of this package
-# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>, soulofdestiny <mgriessmeier@suse.com>
+# Maintainer: soulofdestiny <mgriessmeier@suse.com>
```

Comment:
@Soulofdestiny I assume we should also replace your name here? Do you have a suggestion?